### DISCUSSION TITLE
(Question) How to create a blog list?

### DISCUSSION THREAD

**Discussion by Someonemustnothavethis** [TumblThreeApp/TumblThree#369] [Q&A (Question & Answer)]
There's a button to load a blog list but I don't see anyway to make a blog list, or save the blogs already loaded.   
I'm ashamed to say how many blogs I follow and would like to make several different lists in order to load and update the blogs at different times/dates.
I'm just a user and NOT a programmer (can ya tell?) but I had little problem getting it to work without any instructions.  It's been rock solid stable and saves me shedloads of time.  I use Tumbex exclusively to access the Tumblr blogs.
Thanks.
Bob

**Comment by johanneszab** [comment]
Hi,

there are two possible ways for exporting a list of your loaded blogs:

1. Open settings -> general -> press "export blogs". You can click the browse button to navigate to a different location. The default is a blogs.txt file next to the executable.
2. Select all blogs (ctrl-a)
    Copy (ctrl-c)
    Open a text editor of your choice (e.g. notepad++)
    paste (ctrl-v)

Both ways will create a text file containing a list of the blog/likedby/etc urls.

**Comment by Someonemustnothavethis** [comment]
Thanks for the info, this will making my 'Tumblring' much easier.
Is there a way I can donate say, 20 bucks to the project?  I can't program but I like to support stuff that works.
Bob

**Comment by Someonemustnothavethis** [comment]
Hi Johannes,
   Thanks again for the quick reply.
   As a non-programmer and non-geek, I thought I'd give you some
comments on your
program and this whole Github thing-a-ma-jig.
   I know enough to be able to find my way through a github site and
download programs
but it can be intimidating for the non-techie person.  There is never
a nice big "Download here"
button and documentation, if it exists at all, is geared towards other
programmers and full of
terms few people understand.  As an example, I had to look up
'enqueue'....and I have a science
degree!   I'd love to see this program (among others) go more
mainstream so that it's
accessible to many more people.  Have your Github site for the people
working on it, but also
create a forum for the general user to discuss it and share tips.  
I've been advertising your
program on my Tumblr blog but it would be nice to see it enjoy a wider
audience.
    Can I send you 20 bucks?   I love this thing!
     Bob
On 3/17/2020 at 5:01 AM, "Johannes Meyer zum Alten Borgloh"  wrote:
	Hi, 

	there are two possible ways for exporting a list of your loaded
blogs:  	*Open settings -> general -> press "export blogs". You can
click the browse button to navigate to a different location. The
default is a blogs.txt file next to the executable. 	*Select all blogs
(ctrl-a)
 Copy (ctrl-c)
 Open a text editor of your choice (e.g. notepad++)
 paste (ctrl-v)  

	Both ways will create a text file containing a list of the
blog/likedby/etc urls.

**Comment by thomas694** [comment]
Thanks for your feedback, Bob.

For us developers it's sometimes hard to describe something with the best possible words because we have created it and know how it works, also we don't like to describe it too broad and in too easy words to not make someone feel uncomfortable. So feedback from someone with another point of view is helpful.
Also it is not a difficulty for us to set up a forum, the problem is we do not have time to maintain it due to work load and lack of time.

We need (also) some non-programmers who volunteer to help our project.
